manual/toggl/tools/chocolateyInstall.ps1

```diff
@@ -1 +1 @@
-﻿Install-ChocolateyPackage 'toggl' 'msi' '/quiet' 'https://toggl.com/api/v8/installer?app=td&platform=windows&channel=stable'
+﻿Install-ChocolateyPackage 'toggl' 'exe' '/quiet' 'https://toggl.com/api/v8/installer?app=td&platform=windows&channel=stable'

```

Comment:
I don't believe so in my original testing, but let me pack and test this change to verify; I'll post back when done or update.